crates/hc_sandbox/src/calls.rs

```diff
@@ -301,46 +294,31 @@ pub async fn call(
         force_admin_ports
     };
 
-    let cmds = if running.is_empty() {
+    let admin_clients = if running.is_empty() {
         let paths = if existing.is_empty() {
             crate::save::load(std::env::current_dir()?)?
         } else {
             existing.load()?
         };
         let ports = get_admin_ports(paths.clone()).await?;
-        let mut cmds = Vec::with_capacity(ports.len());
+        let mut clients = Vec::with_capacity(ports.len());
         for (port, path) in ports.into_iter().zip(paths.into_iter()) {
-            match CmdRunner::try_new(port).await {
-                Ok(cmd) => cmds.push((cmd, None, None)),
-                Err(WebsocketError::Io(e)) => {
-                    if let std::io::ErrorKind::ConnectionRefused
-                    | std::io::ErrorKind::AddrNotAvailable = e.kind()
-                    {
-                        let (port, holochain, lair) = run_async(
-                            holochain_path,
-                            ConfigRootPath::from(path),
-                            None,
-                            structured.clone(),
-                        )
-                        .await?;
-                        cmds.push((CmdRunner::new(port).await, Some(holochain), Some(lair)));
-                        continue;
-                    }
-                    bail!(
-                        "Failed to connect to running conductor or start one {:?}",
-                        e
-                    )
-                }
-                Err(e) => {
-                    bail!(
-                        "Failed to connect to running conductor or start one {:?}",
-                        e
+            match AdminWebsocket::connect(format!("localhost:{port}")).await {
+                Ok(client) => clients.push(client),
+                Err(_) => {
```

Comment:
At least log the error?

Reply:
Like so? 410bd04013de70bb60fef18efa121dad87f0d41a